### ISSUE TITLE
Job event `status` is not triggered

### ISSUE THREAD

**Issue by defra91** [veny/GearmaNode#47]
I am trying to use the following:

```
var job = client.submitJob('functionName', payload);

job.on('status', function(status) {
  console.log('STATUS', status);
});
```

But it never enter in the callback and so never prints 'STATUS'.

**Comment by veny** [comment]
you can see all the triggers for that event here: https://github.com/veny/GearmaNode#job-events

did you use your code in one of those contexts?

**Comment by defra91** [comment]
Yes I added the listener for different events, as specified in the documentation. However `status` event is never triggered.
